khelo/lib/ui/flow/matches/add_match/components/over_detail_view.dart

```diff
@@ -0,0 +1,217 @@
+    return OnTapScale(
+      onTap: onTap,
+      enabled: onTap != null,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(vertical: 12.0),
+        child: Row(
+          children: [
+            Expanded(
+              child: Text(
+                title,
+                style: AppTextStyle.subtitle3
+                    .copyWith(color: context.colorScheme.textPrimary),
+              ),
+            ),
+            trailing
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _powerPlayCount(
+    BuildContext context,
+    AddMatchViewNotifier notifier,
+    AddMatchViewState state,
+  ) {
+    return Text(
+      _getPowerPlayCount(state).toString(),
+      style: AppTextStyle.subtitle1
+          .copyWith(color: context.colorScheme.textPrimary),
+    );
+  }
+
+  int _getPowerPlayCount(AddMatchViewState state) {
+    final count = [
+      state.firstPowerPlay,
+      state.secondPowerPlay,
+      state.thirdPowerPlay
+    ].where((play) => play != null && play.isNotEmpty).length;
+    return count;
+  }
+
+  Widget _stepperView(
+    BuildContext context,
+    TextEditingController controller, {
+    required Function() onChanged,
+    required Function(bool) onStepperActionTap,
+  }) {
+    return Container(
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(30),
+        color: context.colorScheme.containerLowOnSurface,
+      ),
+      child: Row(
+        children: [
+          _stepperActionButton(
+            context,
+            isPlus: false,
+            isEnabled: (int.tryParse(controller.text) ?? 0) > 1,
+            onTap: () => onStepperActionTap(false),
+          ),
+          Material(
+            color: Colors.transparent,
+            child: IntrinsicWidth(
+              child: TextField(
+                controller: controller,
+                textAlign: TextAlign.center,
+                maxLength: 3,
+                style: AppTextStyle.header4
+                    .copyWith(color: context.colorScheme.primary),
+                inputFormatters: [
+                  FilteringTextInputFormatter.allow(RegExp(r'[0-9]')),
+                ],
+                decoration: InputDecoration(
+                    filled: true,
+                    fillColor: Colors.transparent,
+                    contentPadding: const EdgeInsets.symmetric(horizontal: 10),
+                    focusedBorder: InputBorder.none,
+                    disabledBorder: InputBorder.none,
+                    enabledBorder: InputBorder.none,
+                    border: InputBorder.none,
+                    hintText: "0",
+                    hintStyle: AppTextStyle.header4
+                        .copyWith(color: context.colorScheme.primary),
+                    counterText: ""),
+                onChanged: (value) => onChanged(),
+                onTapOutside: (event) =>
+                    FocusManager.instance.primaryFocus?.unfocus(),
+              ),
+            ),
+          ),
```

Comment:
I think we need to direct show text count instead if textfield because we are provide increase and decrease option.

Reply:
I don't see it optimal as if over Number is higher, user is not going to tap increment button that much time, suppose if over_number is 100, it should have textfield so use can type manually if over count is high.